### ISSUE TITLE
Second user of the demo app from different browser sessions gets message “Not authorized to access this proxy”

### ISSUE THREAD

**Issue by tabasi** [openanalytics/shinyproxy#85]
Hi
I have installed shiny-proxy 2.0.2 as per the instructions on the website on a Windows 10 host.
I have run into a bug and the steps to reproduce are as follows:
```

1- open two different sessions in two browser windows one normal, one incognito
2- browse to the shiny-proxy demo app on both session and login the same user, for example jack
3- run the hello-world app in one session
4- run the same hellop-world app in the other session
5- you will get the “Not authorized to access this proxy” error
```
[Others seems to have similar issues](https://support.openanalytics.eu/t/second-user-of-app-gets-message-not-authorized-to-access-this-proxy/702) reported on the support forum.

**Comment by dylancis** [comment]
same issue here.

**Comment by fmichielssen** [comment]
There appear to be 2 ways to trigger the issue:

1. An admin accesses an app that is already running for another user.
2. The same user accesses the same app from two different browser sessions.

We are preparing a fix and will release it asap.

**Comment by tabasi** [comment]
I do believe this has been resolved in 2.0.3. 
Thanks for prompt fix guys.
